rainier-core/src/main/scala/com/stripe/rainier/ir/ApplyMethodGenerator.scala

```diff
@@ -5,12 +5,13 @@ final private case class ApplyMethodGenerator(classPrefix: String,
                                               outputMethods: Seq[Int])
     extends MethodGenerator {
   val isStatic: Boolean = false
-  val methodName: String = "apply"
-  val methodDesc: String = "([D[D[D)V"
+  val methodName: String = "output"
```

Comment:
I guess we should rename this class to OutputMethodGenerator

Reply:
+1